target/linux/ar71xx/config-4.4

```diff
@@ -47,6 +47,7 @@ CONFIG_ATH79_MACH_AP136=y
 CONFIG_ATH79_MACH_AP143=y
 CONFIG_ATH79_MACH_AP147=y
 CONFIG_ATH79_MACH_AP152=y
+CONFIG_ATH79_MACH_AP531B0=y
```

Comment:
Dear pepe2k, 
i double check that in my env, original attributes of config-4.4 is 664, not 644.  is there something wrong?

drwxrwxr-x  9 yang yang  4096 Mar  5 11:43 ./
drwxrwxr-x 47 yang yang  4096 Mar  5 06:03 ../
-rw-rw-r--  1 yang yang   493 Mar  5 06:03 Makefile
drwxrwxr-x  4 yang yang  4096 Mar  5 06:03 base-files/
-rw-rw-r--  1 yang yang 12914 Mar  5 11:43 config-4.4
drwxrwxr-x  5 yang yang  4096 Mar  5 06:03 files/
drwxrwxr-x  3 yang yang  4096 Mar  5 06:03 generic/
drwxrwxr-x  3 yang yang  4096 Mar  5 11:43 image/
drwxrwxr-x  3 yang yang  4096 Mar  5 06:03 mikrotik/
-rw-rw-r--  1 yang yang  1837 Mar  5 06:03 modules.mk
drwxrwxr-x  3 yang yang  4096 Mar  5 06:03 nand/
drwxrwxr-x  2 yang yang 12288 Mar  5 06:03 patches-4.4/